integration-test/src/it/relocation-group/verify.groovy

```diff
@@ -46,3 +46,13 @@ assert profile1.size() != 0
 dependency = pom.dependencyManagement.dependencies.dependency.find { it.artifactId.text() == "junit-dep" }
 assert dependency != null
 assert dependency.version.text() == "4.1"
+
+
+def passed = false
+pom.properties.each {
+    if ( it.text().contains ("slf4j") )
```

Comment:
To make it a little clearer, are we looking for "org.slf4j" specifically? I would make it more specific only because both the groupId and artifactId contain "slf4j" and to me it's just a tiny bit clearer which part we are checking for.

Reply:
It to verify that `slf4j` gets substituted into `<slf4j>slf10</slf4j>`

Reply:
I thought it was `slf10 --> org.slf4j`? That's why I wasn't 100% clear there based on the test (for me).

Reply:
Yeah it is, I'll change it.